docs/dense-retrieval.md

```diff
@@ -76,3 +76,80 @@ recall_1000           	all	0.9637
 ```
 
 You'll notice that hnsw index leads to a small loss in effectiveness.
+
+### Hybrid Dense-Sparse Ranking
+MS MARCO passage ranking task, 
+Hybrid
+- dense retrieval with TCT-ColBERT, HNSW index.
+- sparse retrieval with doc2query-T5 expanded index.
+
+```
+python -m pyserini.hsearch --topics msmarco_passage_dev_subset \
+                             --dindex msmarco-passage-tct_colbert-hnsw \
+                             --sindex msmarco-passage-expanded \
+                             --encoded-queries msmarco-passage-dev-subset-tct_colbert \
+                             --output runs/run.msmarco-passage.tct_colbert.hnsw.doc2queryT5.tsv \
+                             --msmarco \
```

Comment:
Should we set batch size here?

Reply:
`batch_search` & `simple_search` both gives an acceptable result, I was trying to do one without `batch_search`. But we can set the batch, which will make it faster than `simple_search`. What do you think @lintool ?

Reply:
See my comment below -